client/src/js/services/FinancialService.js

```diff
@@ -8,33 +8,25 @@ function FinancialService ($http, util) {
 
   service.readCostCenter = readCostCenter;
   service.readProfitCenter = readProfitCenter;
-  service.getCost = getCost;
-  service.getProfit = getProfit;
+  service.getFeeValue = getFeeValue;
 
   // this will read all cost centers
   function readCostCenter(id, params) {
-     var url = '/cost_centers/';
+     var url = '/fee_centers?is_cost=1';
```

Comment:
This is old code so it's not a necessary fix, but you could also just extend the `params` with `{ is_cost: 1}`.  Like this:

```js
var baseUrl = '/fee_centers';

function readCenter(id, params) {
  return $http.get(baseUrl, { params : params })
    .then(util.unwrapHttpResponse);
}

function readProfitCenter(id, params) {
  var profitParameters = angular.extend(params, { is_cost : 0 });
  return readCenter(id, profitParameters);
}

function readCostCenter(id, params) {
  var costParameters = angular.extend(params, { is_cost: 1 });
  return readCenter(id, costParameters);
}
```